pandas/tests/window/test_groupby.py

```diff
@@ -101,6 +107,9 @@ def test_rolling_quantile(self, interpolation):
         g = self.frame.groupby("A")
         r = g.rolling(window=4)
         result = r.quantile(0.4, interpolation=interpolation)
+        # GH 39732
+        g.mutated = True
```

Comment:
why are you setting all of these? what is the purpose?

Reply:
To get the resulting index of `g.apply(lambda x: x.rolling(...)` to match `self.frame..groupby(...).rolling(...)`, the `mutated` attributes on `g` need to be set to `True`

It directly why https://github.com/pandas-dev/pandas/issues/39732 was a bug

Reply:
I will change the expected indexes instead of setting this attribute